src/api/app/models/obs_factory/distribution_strategy_opensuse.rb

```diff
@@ -13,7 +13,9 @@ def opensuse_leap_version
     end
 
     def openqa_version
-      opensuse_leap_version
+      # Only use major version to find the openSUSE Leap job group since we use
+      # the same job group for the whole codestream
+      opensuse_leap_version[0..2]
```

Comment:
I would write a model test for this method.

Similar to [this one](https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/models/obs_factory
/distribution_strategy_sle15_spec.rb) but in /spec/models/obs_factory
/distribution_strategy_opensuse_spec.rb